resources/quality/smoothie_dual/smoothie_0.4_fast.inst.cfg

```diff
@@ -0,0 +1,24 @@
+[general]
```

Comment:
These quality profiles are really specific to the Mosquito Magnum nozzles, and probably also to your specific printer. I'd prefer you to stick with the defaults. This printer should probably not have any specific quality profiles but just use the global ones.

Reply:
removed the quality profiles...